### PR TITLE
Create `biologic-mpr` and `biologic-mpt` filetypes.

### PR DESCRIPTION
**Pull request by PeterKraus**
@ml-evs This is how I'd commit a new bunch of filetypes.

Includes the binary filetype `biologic-mpr` and text export `biologic-mpt`. The `associated_instruments` list is populated from EC Lab.

### CODE REVIEW

File: marda_registry/data/filetypes/biologic-mpr.yml [hunk 1]
```diff
@@ -1,10 +1,38 @@
...
+    BioLogic MPR file
+description: >-
+    A binary data file created by BioLogic cyclers, containing time- and frequency-domain
+    data pertaining to many electrochemical techniques, such as voltammetry,
+    potentiometry, coulometry and impedance spectroscopy.
 subject:
     - electrochemistry
     - voltammetry
-    - impedance spectroscopy
+    - impedance spectropscopy
```
Comment (PeterKraus) [inline marda_registry/data/filetypes/biologic-mpr.yml:13]:
```suggestion
    - impedance spectroscopy
```

File: marda_registry/data/filetypes/biologic-mpt.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+description: >-
+    A text data file created by BioLogic EC-Lab from data from BioLogic cyclers, containing
+    time- and frequency-domain data pertaining to many electrochemical techniques, such as
+    voltammetry, potentiometry, coulometry and impedance spectroscopy.
+associated_vendors:
+    - BioLogic
+subject:
+    - electrochemistry
+    - voltammetry
+    - impedance spectropscopy
```
Comment (PeterKraus) [inline marda_registry/data/filetypes/biologic-mpt.yml:15]:
```suggestion
    - impedance spectroscopy
```

**Review by PeterKraus — COMMENTED** [review]
I can't see what the linter is complaining about. I thought ci-pre-commit might kick in. I'll commit the above two suggestions and see what happens.

**Comment by ml-evs** [comment]
> I can't see what the linter is complaining about. I thought ci-pre-commit might kick in. I'll commit the above two suggestions and see what happens.

I haven't actually enabled it on this repo, but I can. Linting just looks like file endings are getting mangled somehow.

File: marda_registry/data/filetypes/biologic-mpr.yml [hunk 3]
```diff
@@ -1,10 +1,38 @@
...
-associated_vendors:
-    - BioLogic
+id: >-
+    biologic-mpr
+name: >-
+    BioLogic MPR file
+description: >-
+    A binary data file created by BioLogic cyclers, containing time- and frequency-domain
+    data pertaining to many electrochemical techniques, such as voltammetry,
+    potentiometry, coulometry and impedance spectroscopy.
```
Comment (ml-evs) [inline marda_registry/data/filetypes/biologic-mpr.yml:9]:
```suggestion
    potentiometry, coulometry and impedance spectroscopy.
associated_vendors:
    - BioLogic
```

Sorry, think I mangled this one myself on the merge

File: marda_registry/data/filetypes/biologic-mpt.yml [hunk 4]
```diff
@@ -0,0 +1,41 @@
+---
+id: >-
+    biologic-mpt
+name: >-
+    BioLogic MPT file
+description: >-
+    A text data file created by BioLogic EC-Lab from data from BioLogic cyclers, containing
+    time- and frequency-domain data pertaining to many electrochemical techniques,
+    such as
+    voltammetry, potentiometry, coulometry and impedance spectroscopy.
```
Comment (ml-evs) [inline marda_registry/data/filetypes/biologic-mpt.yml:10]: some strange ideas about line length from pre-commit here... not a priority but can try to fix it at the repo level after this PR

**Review by ml-evs — APPROVED** [review]
LGTM!

**Pull request merged**